src/Control/Biegunka/Source/Git/Internal.hs

```diff
@@ -9,24 +9,24 @@ module Control.Biegunka.Source.Git.Internal
   , Git (..)
   , actions, branch, failIfAhead
   , URI
-  , askGit, askGit_, updateGit
+  , runGit, updateGit
   , gitHash
   , defaultGit
   ) where
 
-import           Control.Monad                      (void, when)
```

Comment:
Next time please prevent me from `stylish-haskell`ify imports if this is a format you want to keep them with